lib/BuildSystem/SwiftTools.cpp

```diff
@@ -288,13 +294,17 @@ class SwiftCompilerShellCommand : public ExternalCommand {
       importPaths = std::vector<std::string>(imports.begin(), imports.end());
     } else if (name == "temps-path") {
       tempsPath = value;
-    } else if (name == "is-library") {
+    } else if (name == "is-library" || name == "enable-wmo") {
       if (value != "true" && value != "false") {
         ctx.error("invalid value: '" + value + "' for attribute '" +
                   name + "'");
         return false;
       }
-      isLibrary = value == "true";
+      bool boolValue = value == "true";
+      if (name == "is-library")
+        isLibrary = boolValue;
+      else if (name == "enable-wmo")
+        enableWholeModuleOptimization = boolValue;
```

Comment:
I would rather factor out a helper function for bool reading than lump these into one `else if`, I find that less readable.